src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.xml

```diff
@@ -5,6 +5,8 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
+        <service id="api_platform.doctrine.query_name_generator" class="ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGenerator" public="false" />
```

Comment:
This class contains states, so it's not a service (it should be instantiated directly in data providers).